content/articles/object-oriented-programming-in-python/index.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /object-oriented-programming-in-python/
+title: Object Oriented Programming in Python
+description: This article covers the basics of the object oriented concepts in the Python programming language. We will create a simple class to demonstrate these concepts.
+author: joseph-abiola
+date: 2021-11-01T00:00:00-13:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/object-oriented-programming-in-python/hero.jpg
+    alt: Object Oriented Programming Python Image Hero
+---
+
+For the last four years, security researchers have reported a tremendous increase in security breaches. About 20-30,000 websites get hacked every day. There are several ways websites get hacked, XML External Entity Attacks (XXE) being one of them.

```

Comment:
@mercymeave I believe, you copied the wrong description in the front matter. Am I right?